src/util/logger.cpp

```diff
@@ -0,0 +1,36 @@
+#include <cstring>
+
+#include "util/logger.h"
+
+
+namespace {
+
+const char* const kLogPreambleSuffix = " -";
+const std::size_t kLogPreambleSuffixLen = std::strlen(kLogPreambleSuffix);
```

Comment:
if you use here a QLatin1String, the size() member can be used without cost. Alternative, a constexp may work.